Scripts/script-CreateEmailHtmlBody.yml

```diff
@@ -11,15 +11,15 @@ script: |
   // get a specific label from incident labels
   var getLabel = function(incident,path) {
       label = path.split('.')[2];
-      for (var i=0;i<incident.labels.length;i++) {
-          if (incident.labels[i].type==label) {
+      for (var i=0; i<incident.labels.length; i++) {
```

Comment:
understand the urge to fix formatting, but this makes the reviews a bit more challenging (also for customers who might want to review diff)... recommend next time avoiding :)